constants.js

```diff
@@ -14,6 +14,9 @@ const SYNC_IN_PROGRESS = 'Last Sync: In progress';
 const SYNC_SUCCESSFUL = 'Last Sync: Successful';
 const SYNC_FAILED = 'Last Sync: Failed';
 const DISABLED = 'disabled';
+const STATUS_BASE_URL_PROD = 'https://status.realdevsquad.com';
+const STATUS_BASE_URL_STAGING = 'https://staging-status.realdevsquad.com';
```

Comment:
Where we are using this URL?

Reply:
<img width="240" alt="Screenshot 2023-08-10 at 7 56 19 AM" src="https://github.com/Real-Dev-Squad/website-dashboard/assets/98796547/c9a5f71e-e929-4510-9a52-5a3bbea86051">

To redirect to status site. click [here](https://github.com/Real-Dev-Squad/website-dashboard/pull/446/files#diff-699a12f3b559d9521daf69ab59f425bfcd272e2cb154c288b0b4517dd9da85e1R460) for code

Reply:
Not necessary now, Will use it in subsequent PRs